src/ngraph/op/and.cpp

```diff
@@ -19,14 +19,16 @@
 using namespace std;
 using namespace ngraph;
 
-op::And::And(const shared_ptr<Node>& arg0, const shared_ptr<Node>& arg1)
-    : BinaryElementwiseLogical("And", arg0, arg1)
+op::And::And(const shared_ptr<Node>& arg0,
+             const shared_ptr<Node>& arg1,
+             const AutoBroadcastSpec& autob)
+    : BinaryElementwiseLogical("And", arg0, arg1, autob)
```

Comment:
Do we need to pass this to every constructor ? I assume autob mode should be fixed for all nodes during at least on ngraph invocation. Can't we instead implicitly read it from some configuration class/global  ?

Reply:
Provide a way to change the default constructor of AutoBroadcastSpec?

Reply:
It might make sense to have a global that influences the constructors' behavior for the sake of convenience, but I still think we'll need to store `autob` and respect that stored value, for things like deserialization.

Reply:
Basically, is it possible that nodes may have different auto-broadcast modes ? If not, then makes more sense to keep it as a global.

Reply:
Well, "possible" is up to us, right? ;)

Consider, for example, a builder (either one that's part of Core and meant to be reused across bridges, or one that's used internally by the bridge) that wants to use autobroadcast internally. How would that interact with the global?